### PR TITLE
Elderly at home national page

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard10**
## Summary

Set up "elderly at home" national page.

### CODE REVIEW

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: src/pages/landelijk/thuiswonende-ouderen.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+          value: value.positive_tested_daily,
+          date: value.date_of_report_unix,
+        }))}
+        metadata={{ source: text.section_positive_tested.bronnen.rivm }}
+      />
+
+      <ChoroplethTile
+        title={text.section_positive_tested.choropleth_daily_title}
+        description={text.section_positive_tested.choropleth_daily_description}
+        metadata={{ source: text.section_positive_tested.bronnen.rivm }}
```
Comment (VWSCoronaDashboard4) [inline src/pages/landelijk/thuiswonende-ouderen.tsx:95]: I guess this metadata should have a date as well?

File: src/locale/nl.json [hunk 2]
```diff
@@ -129,6 +129,73 @@
       }
     }
   },
+
+  "thuiswonende_ouderen": {
+    "titel_sidebar": "Thuiswonende ouderen",
+    "titel_kpi": "Patiënten met eerste melding van COVID-19 klachten bij de huisarts",
+
```
Comment (VWSCoronaDashboard4) [inline src/locale/nl.json:136]: The other copy does not use blank lines. Not sure if lokalize will just remove them though?

File: src/pages/landelijk/thuiswonende-ouderen.tsx (hunk 1 above)
Comment (VWSCoronaDashboard10) [reply src/pages/landelijk/thuiswonende-ouderen.tsx:95]: Yep, I guess you're right

File: src/locale/nl.json (hunk 2 above)
Comment (VWSCoronaDashboard10) [reply src/locale/nl.json:136]: Might be, don't think that's a big problem.

**Pull request merged**